### PR TITLE
fix(select-status): #CO-1782 remove when you click on edit ticket it will change status to open

### PR DESCRIPTION
**Pull request by Ozen40**
## Describe your changes
As an adml when you click on edit ticket button, it will not change the status from new to open
## Issue ticket number and link
[CO-1782](https://jira.support-ent.fr/browse/CO-1782)
## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [ ] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: src/main/resources/public/ts/controllers/controller.ts [hunk 1]
```diff
@@ -310,9 +310,7 @@ export const SupportController: Controller = ng.controller('SupportController',
 		};
 
 		$scope.preSelectNewTicketStatus = async (): Promise<void> => {
-			if ($scope.userIsLocalAdmin($scope.ticket) && $scope.ticket.status == model.ticketStatusEnum.NEW) {
-				$scope.editedTicket.status = model.ticketStatusEnum.OPENED
-			} else if ($scope.userIsLocalAdmin($scope.ticket) && $scope.ticket.status == model.ticketStatusEnum.OPENED) {
+			if ($scope.userIsLocalAdmin($scope.ticket) && $scope.ticket.status == model.ticketStatusEnum.OPENED) {
```
Comment (qpic-cgi) [inline src/main/resources/public/ts/controllers/controller.ts:313]: cette méthode n'est plus utile du coup non ?
car $scope.editedTicket.status = $scope.ticket.status systématiquement maintenant non ?

**Review by qpic-cgi — CHANGES_REQUESTED** [review]

File: src/main/resources/public/ts/controllers/controller.ts (hunk 1 above)
Comment (Ozen40) [reply src/main/resources/public/ts/controllers/controller.ts:313]: Oui mais justement pour le fait que ça ne passe plus de open -> waiting c'est le [CO-1783](https://jira.support-ent.fr/browse/CO-1783)

Comment (qpic-cgi) [reply src/main/resources/public/ts/controllers/controller.ts:313]: okey, à la fin il te restera juste le "ping pong" en mode "OPEN -> WAITING, WAITING -> OPEN... " c'est ça ?

Comment (Ozen40) [reply src/main/resources/public/ts/controllers/controller.ts:313]: Oui sachant que le waiting est mis manuellement par l'adml et le WAITING->OPEN est auto

**Pull request merged**